dialog_test.go

```diff
@@ -298,7 +298,7 @@ func TestOpenDialog(t *testing.T) {
 	}
 	err = api.OpenDialog("", *dialog)
 	if err == nil {
-		t.Errorf("Did not error with empty trigger", err)
```

Comment:
I found this problem when I ran tests locally.

Reply:
Fixed in 7a2193d189e649d76c4a90e274e49b24b725fcc5.